### PR TITLE
First commit of next_day udf

### PR DESCRIPTION
**Pull request by theodoresiu**

### CODE REVIEW

File: third_party/compilerworks/vertica_to_bq_migration/next_day.sql [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ */
+
+
+ /* Returns the date of the first weekday (second arugment) that is later than the date specified by the first argument */
+CREATE OR REPLACE FUNCTION cw_udf.cw_next_day(date_value DATE, day_name STRING) RETURNS DATE AS (
```
Comment (danieldeleo) [inline third_party/compilerworks/vertica_to_bq_migration/next_day.sql:29]:
```suggestion
CREATE OR REPLACE FUNCTION cw_next_day(date_value DATE, day_name STRING) RETURNS DATE AS (
```

File: third_party/compilerworks/vertica_to_bq_migration/README.md [hunk 2]
```diff
@@ -3,8 +3,21 @@
...
+
+
+## Documentation
+
+
+### [next_day(date_value DATE, day_name STRING)](next_day.sql)
+This function calculates the next occurance of a specific day based off an initial date value. The function is based off of the Vertica (function)[https://www.vertica.com/docs/9.2.x/HTML/Content/Authoring/SQLReferenceManual/Functions/Date-Time/NEXT_DAY.htm] 
+```sql
+   SELECT bqutil.vertica_to_bq.next_day(Date('2016-04-29'),'mon'); 
+   SELECT bqutil.vertica_to_bq.next_day(Date('2016-04-29'),'fri');
```
Comment (danieldeleo) [inline third_party/compilerworks/vertica_to_bq_migration/README.md:20]: I don't believe we'll be hosting these UDFs in our bqutil project. @ryanmcdowell thoughts?

File: third_party/compilerworks/vertica_to_bq_migration/next_day.sql (hunk 1 above)
Comment (theodoresiu) [reply third_party/compilerworks/vertica_to_bq_migration/next_day.sql]: Fixed

File: third_party/compilerworks/migration/common/next_day.sql [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+       WHEN 'thu' THEN 5
+       WHEN 'fri' THEN 6
+       WHEN 'sat' THEN 7
+       ELSE NULL END AS tgt,
+       extract(dayofweek FROM date_value) AS src
+    ) SELECT CASE WHEN src < tgt
+             THEN date_add(date_value, INTERVAL (tgt - src) DAY)
+             ELSE date_add(date_value, INTERVAL (tgt + 7 - src) DAY) END
+             FROM t)
+);
```
Comment (danieldeleo) [inline third_party/compilerworks/migration/common/next_day.sql:45]:
```suggestion
);

```

**Review by danieldeleo — APPROVED** [review]

**Pull request merged**